src/main/java/io/github/noeppi_noeppi/mods/bongo/compat/JadeIntegration.java

```diff
@@ -0,0 +1,40 @@
+package io.github.noeppi_noeppi.mods.bongo.compat;
+
+import io.github.noeppi_noeppi.mods.bongo.Bongo;
+import io.github.noeppi_noeppi.mods.bongo.BongoMod;
+import io.github.noeppi_noeppi.mods.bongo.util.ItemRenderUtil;
+import mcp.mobius.waila.api.*;
+import mcp.mobius.waila.api.config.IPluginConfig;
+import net.minecraft.client.Minecraft;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.block.Block;
+
+@WailaPlugin
+public class JadeIntegration implements IWailaPlugin, IComponentProvider {
```

Comment:
`WailaIntegration` as it is not really jade specific and should work with other WAILA forks as well